recipes/bioconductor-motifcounter/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.2.1" %}
+{% set name = "motifcounter" %}
+{% set bioc = "3.6" %}
+
+package:
+  name: 'bioconductor-{{ name|lower }}'
+  version: '{{ version }}'
+source:
+  fn: '{{ name }}_{{ version }}.tar.gz'
+  url:
+    - 'http://bioconductor.org/packages/{{ bioc }}/bioc/src/contrib/{{ name }}_{{ version }}.tar.gz'
+    - 'https://bioarchive.galaxyproject.org/{{ name }}_{{ version }}.tar.gz'
+    - 'https://depot.galaxyproject.org/software/bioconductor-{{ name }}/bioconductor-{{ name }}_{{ version }}_src_all.tar.gz'
+  sha256: 80ad9a1c25eb7acb0fcdf55fea365a5e8a4c3084aa33c2cf043c151e14f387cc
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - bioconductor-biostrings
+    - r-base
+    - gcc  # [linux]
+    - llvm  # [osx]
+  run:
+    - bioconductor-biostrings
+    - r-base
```

Comment:
`libgcc  # [linux]` is missing here. @wkopp have you used the bioconductor skeleton generator?

Reply:
yes, I've used the bioconductor skeleton generator from bioconda-utils.